PathPlanning/DStarLite/DStarLite.py

```diff
@@ -0,0 +1,278 @@
+"""
+D* Lite grid planning
+author: vss2sn (28676655+vss2sn@users.noreply.github.com)
+Link to papers:
+D* Lite (Link: http://idm-lab.org/bib/abstracts/papers/aaai02b.pd)
+Improved Fast Replanning for Robot Navigation in Unknown Terrain
+(Link: http://www.cs.cmu.edu/~maxim/files/dlite_icra02.pdf)
+Implemented maintaining similarity with the pseudocode for understanding
+"""
+from __future__ import annotations
```

Comment:
This project only support Python 3.9. So, maybe this import is not needed.

Reply:
That was left as Python 3.8 is currently the default for Ubuntu, but will modify.